### PR TITLE
Document precedence of route definitions

### PR DESCRIPTION
**Pull request by aryzing**
Adds small section on route matching precedence.

### CODE REVIEW

File: docs/src/network.md [hunk 1]
```diff
@@ -421,6 +421,17 @@ await page.RouteAsync("**/api/fetch_data", async route => {
...
+#### Precedence
+
+When a request matches multiple routes, the last matched route is used to handle the request.
+
+```js
+page.route("**/api", route => route.abort());
+page.route("**/api/useful-resource", route => route.fulfill(/* data */));
+```
+
+The above routes will return data for `useful-resource`, and block all other requests to the API.
```
Comment (pavelfeldman) [inline docs/src/network.md:433]: Do you want to also mention the `route.fallback` here?

Comment (aryzing) [reply docs/src/network.md]: Great idea. Added, and reworded the text a little. 

Not sure the link is formatted correctly, I looked at how other links are defined and tried to follow the same convention, let me know if it needs changing.

File: docs/src/network.md [hunk 2]
```diff
@@ -421,6 +421,17 @@ await page.RouteAsync("**/api/fetch_data", async route => {
 await page.GotoAsync("https://example.com");
 ```
 
+#### Precedence
+
+When a request matches multiple routes, the last matched route is used to handle the request. As an example, the routes below will return data for `useful-resource`, and block all other requests to the API.
+
+```js
```
Comment (pavelfeldman) [inline docs/src/network.md:428]: Sorry for the slow review. Now we need the snippets for the other languages as well, as below.

Comment (aryzing) [reply docs/src/network.md:428]: Thanks for the review. I'm not familiar with the other languages, so I'll close the issue for now.

**Pull request closed without merging**